x-pack/heartbeat/monitors/browser/source/source.go

```diff
@@ -1,6 +1,12 @@
 // Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
+<<<<<<< HEAD
+=======
+
+//go:build linux || darwin
+// +build linux darwin
+>>>>>>> 8552e34a6a (ci: enable windows for testing heartbeat (#32937))
```

Comment:
@andrewvc , is this correct?

```suggestion

//go:build linux || darwin
// +build linux darwin
```